include limits.h to avoid UINT_MAX undefined compiling error.
remove the unused assign which caused compiling error with -Werror.

Upstream-Status: Pending

Signed-off-by: Yao Zhao <yao.zhao@windriver.com>
---

diff --git a/usl/usl_timer.c b/usl/usl_timer.c
index 734b820..fda752b 100644
--- a/usl/usl_timer.c
+++ b/usl/usl_timer.c
@@ -42,6 +42,7 @@
 #include <signal.h>
 #include <string.h>
 #include <sys/time.h>
+#include <limits.h>
 
 #include "usl.h"
 
@@ -87,14 +88,13 @@ void (*usl_timer_tick_hook)(void);
  */
 void usl_timer_tick(void)
 {
-	int result;
 	char msg = '\0';
 
 	usl_tick++;
 
 	if (!usl_tick_pending) {
 		usl_tick_pending = 1;
-		result = write(usl_tick_pipe[1], &msg, sizeof(msg));
+		write(usl_tick_pipe[1], &msg, sizeof(msg));
 	}
 }
 
@@ -111,11 +111,10 @@ static void usl_timer_tick_handler(int fd, void *arg)
 	struct usl_ord_list_head *tmp;
 	struct usl_list_head *iwalk;
 	struct usl_list_head *itmp;
-	int result;
 	char msg;
 	USL_LIST_HEAD(expire_list);
 
-	result = usl_fd_read(usl_tick_pipe[0], &msg, sizeof(msg));
+	usl_fd_read(usl_tick_pipe[0], &msg, sizeof(msg));
 	usl_tick_pending = 0;
 
 	usl_list_for_each(walk, tmp, &usl_timer_list) {
